generator/sig_readme.tmpl

```diff
@@ -25,9 +25,9 @@ Meeting recordings can be found [here]({{.MeetingRecordingsURL}}).
 {{- if .Label }}
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2F{{.Label}})
 {{- end }}
-{{if .Contact.FullGitHubTeams}}
-## GitHub Teams
-{{range .Contact.GithubTeamNames -}}
-* [@{{.}}](https://github.com/kubernetes/teams/{{.}})
+{{if .Contact.GithubTeams}}
+### GitHub Teams
+{{range .Contact.GithubTeams -}}
```

Comment:
Can people who are not part of the K8s org access these links? I remember seeing 404s before I joined. If so, maybe we should add a sentence explaining that non-members cannot access them to avoid confusion?

Reply:
This is also what I concern about :)